shopinvader_assortment/demo/shopinvader_assortment_demo.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2018 ACSONE SA/NV
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+    <record id="shopinvader_assortment1" model="ir.filters">
+        <field name="name">Demo Shopinvader Backend</field>
+        <field name="model_id">product.product</field>
+        <field name="domain">[('sale_ok', '=', True)]</field>
+        <field name="user_id" eval="False"/>
+        <field name="is_assortment" eval="True"/>
```

Comment:
Is this field `is_assortment` defined on `ir.filter`?

Reply:
yes in product_assortment addon

Reply:
indeed I answered to the my comment in an other comment .... https://github.com/akretion/odoo-shopinvader/pull/216/files#r219508550